fastlane_core/lib/fastlane_core/ios_app_identifier_guesser.rb

```diff
@@ -0,0 +1,57 @@
+module FastlaneCore
+  class IOSAppIdentifierGuesser
+    class << self
+      def guess_app_identifier_from_args(args)
+        # args example: ["-a", "com.krausefx.app", "--team_id", "5AA97AAHK2"]
+        args.each_with_index do |current, index|
+          if current == "-a" || current == "--app_identifier"
+            return args[index + 1] if args.count > index
```

Comment:
The `args.count` will avoid an index out of bounds, right?

Reply:
Yeah!